src/gp_instance.hpp

```diff
@@ -35,16 +58,70 @@ class GPInstance {
   // and PCSS bitsets are at the end.
   // The collection of rootsplits, with the same indexing as in the indexer_.
   BitsetVector rootsplits_;
-  // A map going from the index of a PCSS to its child.
+  // A map going from the index of a PCSP to its child.
   SizeBitsetMap index_to_child_;
   // A map going from a parent subsplit to the range of indices in
   // sbn_parameters_ with its children. See the definition of Range for the indexing
   // convention.
   BitsetSizePairMap parent_to_range_;
 
+  // The first entries are reserved for fake subsplits.
+  // We will adopt the convention of denoting a trivial subsplit by placing
+  // 0's in the first chunk.
+  // The next set of entries store the rootsplits.
+  // The remaining entries - 1 are for all other subsplits.
+  // The last entry is saved for pseudoroot -- for passing stationary.
+  // The pseudoroot is a Bitset with all 1's.
+  std::unordered_map<Bitset, size_t> subsplit_to_index_;
+  std::vector<std::shared_ptr<DAGNode>> dag_nodes_;
+
+  // Build indexing scheme for q_, branch_lengths_, log_likelihoods_.
+  // PCSP to index.
+  BitsetSizeMap pcsp_indexer_;
+  // Stores range of indices for a subsplit and rotated subsplit.
```

Comment:
So the return value of this map is the concatenation of that for a subsplit and then the rotated subsplit? Given that I mentioned it before, I anticipate that there are reasons to have quite a number of different indexers that are separate than those used by the rest of libsbn, but it'd be nice to justify the difference.